components/Profile/Profile.tsx

```diff
@@ -17,14 +17,14 @@ const Profile: React.FC<ProfileProps> = (props: ProfileProps) => {
 
   return (
     <View style={styles.container}>
-      <Text
+      <HeaderText
         style={[
           styles.greetingHeader,
           { color: Colors[colorScheme].headerColor },
```

Comment:
No need for this color style if you use HeaderText

Reply:
Fixed!